lib/transport/datagouvfr/client.ex

```diff
@@ -3,159 +3,67 @@ defmodule Transport.Datagouvfr.Client do
   An API client for data.gouv.fr
   """
 
-  use HTTPoison.Base
   alias Transport.Datagouvfr.Authentication
+  alias OAuth2.Client, as: OAuth2Client
```

Comment:
Can't we use the same encoding for the four of them ?

Reply:
what do you mean?

Reply:
alias OAuth2.Client, as: OAuth2Client
alias OAuth2.Response, as: OAuth2Response

etc...

Reply:
ah ok, here it's just to avoid confusion with Datagouvfr.Client